ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -124,7 +118,7 @@ final class StandaloneApiServer(
     } yield {
       writePortFile(apiServer.port)
       logger.info(
-        s"Initialized API server version ${BuildInfo.Version} with ledger-id = ${initialConditions.ledgerId}, port = ${apiServer.port}, dar file = ${config.archiveFiles}")
+        s"Initialized API server version ${BuildInfo.Version} with ledger-id = $ledgerId, port = ${apiServer.port}, dar file = ${config.archiveFiles}")
```

Comment:
Not part of this PR, but should we also print more details at a lower log level? Maybe the whole `(ledgerId, config, commandConfig, partyConfig, ledgerConfig)` at DEBUG?